src/demos/components/properties.jsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import PropertyJson from "./components_properties.json";
+import { Grid, Inner, Cell } from "../../../src/libs";
+
+const Head = ["Name", "Default", "Type"];
+
+function Property(Component) {
+  const bodySingle = [];
+  const bodyMultipleProps = [];
+  const PropsValue = [];
+  let count = 0;
+  Object.keys(PropertyJson).forEach((props) => {
+    if (props === Component.name) {
+      if (PropertyJson[props].length === undefined) {
+        Object.keys(PropertyJson[props]).forEach((GeneralPropertyName) => {
+          bodyMultipleProps.push([]);
+          PropertyJson[props][GeneralPropertyName].forEach(element => (
+            bodyMultipleProps[count].push(element)
+          ));
+          PropsValue.push(GeneralPropertyName);
+          count += 1;
+        });
+      } else {
+        PropertyJson[props].forEach(element =>
+          (
+            bodySingle.push(element)
+          ));
+      }
+    }
+  });
+  count = -1;
+  return (
+    (bodySingle.length === 0) ?
+      (bodyMultipleProps.map(element => (
+        <Grid key={element}>
+          <Inner>
+            {/* eslint-disable no-plusplus */}
+            <Cell style={{ fontSize: "20px", fontWeight: "bold" }}>{ PropsValue[count++] }</Cell>
+            {/* eslint-enable no-plusplus */}
+          </Inner>
+          <Inner>
+            {Head.map(title => <Cell key={title}>{title}</Cell>)}
+          </Inner>
+          {element.map(e => (
+            <Inner key={e} style={{ borderBottom: "solid black 1px", marginBottom: "16px", paddingTop: "5px" }}>
+              {e.map(value => <Cell key={value}>{value}</Cell>)}
+            </Inner>
+          ))}
+        </Grid>
+      )))
+      :
+      <Grid>
+        <Inner>
+          {Head.map(title => <Cell key={title}>{title}</Cell>)}
+        </Inner>
+        {bodySingle.map(e => (
+          <Inner key={e} style={{ borderBottom: "solid black 1px", marginBottom: "16px", paddingTop: "5px" }}>
+            {e.map(value => <Cell key={value} >{value}</Cell>)}
+          </Inner>
+        ))}
+      </Grid>
```

Comment:
You should not use the ternary operator because the code is not readable. Here is a better alternative:

```
if (bodySingle.length === 0) {
   return bodyMultipleProps.map( ... );
}

return (
  <Grid>
    ...
  </Grid>
);
```

```